src/trade/tradeConf.es6

```diff
@@ -147,26 +137,16 @@ const register_ticks = (state, extra) => {
       }
    }
 
-   let entry = null, expiry = null;
-   let tracking_timeout_set = false;
-   const track_ticks = () => {
-      tracking_timeout_set = false;
-      last_1000_ticks.filter(
-         (tick) => (tick.symbol === extra.symbol && tick.epoch*1 >= entry && tick.epoch*1 <= expiry)
-      ).forEach(add_tick);
-      if(tick_count > 0) {
-         tracking_timeout_set = true;
-         setTimeout(track_ticks, 300);
-      }
-   }
+   let entry = null,
+      expiry = null;
 
    fn = liveapi.events.on('proposal_open_contract', (data) => {
       const contract = data.proposal_open_contract;
       if(contract.contract_id !== extra.contract_id) return;
       entry = contract.entry_tick_time * 1;
       expiry = contract.date_expiry * 1; /* date_expiry gets updated when contract is settled !!! */
-      if(!tracking_timeout_set)
-         track_ticks();
+      if (contract.current_spot_time >= entry)
+         add_tick(data.proposal_open_contract);
```

Comment:
@arnabk @4p00rv By the time `porposal_open_contract` event is registered there might already be some ticks that have been passed. In other words not all ticks for a contract are returned with this api.

The track_ticks() function and the last_1000_ticks[] array that are now removed are intended to fix this issue.
@arnabk With those stuff gone the trade module will be broken.